drivers/gles2/rasterizer_storage_gles2.cpp

```diff
@@ -3051,6 +3051,7 @@ void RasterizerStorageGLES2::multimesh_instance_set_color(RID p_multimesh, int p
 	ERR_FAIL_COND(!multimesh);
 	ERR_FAIL_INDEX(p_index, multimesh->size);
 	ERR_FAIL_COND(multimesh->color_format == VS::MULTIMESH_COLOR_NONE);
+	ERR_FAIL_INDEX(multimesh->color_format, VS::MULTIMESH_COLOR_FLOAT);
```

Comment:
Now I have this errors 
``` drivers/gles2/rasterizer_storage_gles2.cpp:3087 - Index multimesh->custom_data_format=2 out of size (VisualServer::MULTIMESH_CUSTOM_DATA_FLOAT=2)```

Shouldn't this line look like 
```ERR_FAIL_INDEX(multimesh->color_format, VS::MULTIMESH_COLOR_FLOAT);``` 
or after adding new enum value 
```ERR_FAIL_INDEX(multimesh->color_format, VS::MULTIMESH_COLOR_MAX);``` ?

Reply:
Your error is for line 3087 (below) which is for ``multimesh_set_custom_data()``. 

However, I see where the issue may be. I may have misunderstood when FAIL_INDEX will fail. 

Indeed: It fails when ``index >= size`` I though it failed when ``index > size``. Looks like we do need to add a new enum.
https://github.com/godotengine/godot/blob/f2aa99a8e2d33e534fa3adbff2981a9c902bbf32/core/error_macros.h#L123-L129